libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -690,6 +690,23 @@
         ENCRYPTED_LOGGING_UPLOAD_SUCCESSFUL,
         ENCRYPTED_LOGGING_UPLOAD_FAILED,
         READER_POST_REPORTED,
+        COMMENT_APPROVED,
+        COMMENT_UNAPPROVED,
+        COMMENT_SPAMMED,
+        COMMENT_UNSPAMMED,
+        COMMENT_FOLLOWED,
+        COMMENT_UNFOLLOWED,
+        COMMENT_LIKED,
+        COMMENT_UNLIKED,
+        COMMENT_TRASHED,
+        COMMENT_UNTRASHED,
```

Comment:
Not sure how untrashing a comment should work, trying it on notifications I see that for a trashed comment we have a `Restore` action where we usually have an Approve/Approved button. Using it I get a `comment_approved` event instead of a `comment_untrashed`. Maybe something similar is going on in backend logic like for SPAM/UNSPAM?

Update: in site comments, when trashing there is a snackbar that allows to UNDO the trashing, I think we don't trace that.

Reply:
Think I figured this one out. Regarding the UNDO snackbar, we don't track that one, since the actual action happens when snackbar is dismissed, and undo is more like "I changed my mind" thing :)